CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Use SocketLB on host namespace only.
```

Comment:
It would be nice to explain why we do that I think. I'm 100% sure I will forget soon.

Reply:
hey hey, I linked the upstream issue in the PR description and the commit message has the reasoning behind the change. 

Do you think that I should add more info? In that case, where would be the best place for that?

Reply:
Good enough for me, thanks.